Adyen/Formatters/PhoneNumberFormatter.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// :nodoc:
+open class PhoneNumberFormatter: Adyen.Formatter {
```

Comment:
Are we going to use this in Qiwi payment as well?

Reply:
don't think so, no need to change Qiwi